packages/esm-form-render-app/src/render/forms-render-test.component.tsx

```diff
@@ -74,6 +76,82 @@ function FormRenderTest() {
     setIsSchemaLoaded(false);
   };
 
+  const handleFormValidation = () => {
+    if (defaultJson) {
+      const parsedForm = typeof defaultJson == 'string' ? JSON.parse(defaultJson) : defaultJson;
+
+      for (let i = 0; i < parsedForm.pages.length; i++) {
+        for (let j = 0; j < parsedForm.pages[i].sections.length; j++) {
+          for (let k = 0; k < parsedForm.pages[i].sections[j].questions.length; k++) {
+            const questionObject = parsedForm.pages[i].sections[j].questions[k];
+            handleQuestionValidation(questionObject);
+          }
+        }
+      }
+    } else {
+      console.log('Empty form!');
+    }
+  };
+
+  const handleQuestionValidation = (conceptObject) => {
+    conceptObject.questionOptions.concept
+      ? openmrsFetch(`/ws/rest/v1/concept/${conceptObject.questionOptions.concept}`)
+          .then((response) => {
+            console.log(response.data);
+
+            conceptObject.questionOptions.concept === response.data.uuid
+              ? dataTypeChecker(conceptObject, response)
+              : console.log("❌ response UUID doesn't match concept UUID");
+          })
+          .catch((error) => {
+            if (error.message.includes('500')) {
+              console.log('500 server error occurred!');
+            }
+
+            const clientErrors = ['0', '1', '3', '4'];
```

Comment:
I don't think this is necessary but let's merge this PR since this is urgently needed. We can improve the error handling at a later stage